aten/src/ATen/native/quantized/cpu/q_adaavgpool.cpp

```diff
@@ -296,5 +448,29 @@ Tensor quantized_adaptive_avg_pool2d(
   return output;
 }
 
+Tensor& quantized_adaptive_avg_pool3d_out(
```

Comment:
nit: typical argument order should probably be input, output? why does output come before input in this case?

Reply:
I think the order should probably be: input, output_size, output

Reply:
That what I did initially, but then I saw that out variants for `native_functions` always put output first, so that I did

Reply:
Also, style guides agree that it should be input->output. I will see if it is possible :)

Reply:
I checked -- in this particular case (out variant) the output has to be first. For all other cases I refactored.